src/Employer/Employer.Web/Controllers/DashboardController.cs

```diff
@@ -39,5 +42,16 @@ public async Task<IActionResult> Dashboard([FromRoute]string employerAccountId,[
 
             return View(vm);
         }
+
+        [HttpPost("dismiss-alert", Name = RouteNames.Dashboard_DismissAlert)]
+        public async Task<IActionResult> DismissAlert([FromRoute] string employerAccountId, AlertDismissalEditModel model)
+        {
+            if (Enum.TryParse(typeof(AlertType), model.AlertType, out var alertTypeEnum))
+            {
+                await _orchestrator.DismissAlert(User.ToVacancyUser(), (AlertType)alertTypeEnum);
```

Comment:
If you use generic TryParse as suggested above, there is no need to cast the variable here.

Reply:
👍